src/CMakeLists.txt

```diff
@@ -671,7 +671,7 @@ if (${NUPIC_BUILD_PYEXT_MODULES})
   set(CMAKE_SWIG_FLAGS ${src_swig_flags} ${CMAKE_SWIG_FLAGS})
 
   # Set up linker flags for python extension shared libraries
-  set(src_swig_extension_link_flags "${PYEXT_LINKER_FLAGS_OPTIMIZED}")
+  set(_SRC_SWIG_EXTENSION_LINK_FLAGS "${PYEXT_LINKER_FLAGS_OPTIMIZED}")
```

Comment:
I think I will soon have to bite the bullet and switch "private globals" to leading underscore plus all caps. Having them in lowercase is confusing when you use them inside a function, which itself uses lower-case property names locally.

Here, I started changing the few that I needed inside functions to all-caps.